packages/jest-config/src/normalize.ts

```diff
@@ -478,6 +478,10 @@ export default function normalize(
     options.testRunner = require.resolve('jest-jasmine2');
   }
 
+  if (!options.testSequencer) {
```

Comment:
good catch. Do we need the `if` at all? Or can we do what we do on line 463?

```ts
  options.testEnvironment = getTestEnvironment({
    rootDir: options.rootDir,
    testEnvironment: options.testEnvironment || DEFAULT_CONFIG.testEnvironment,
  });
```

So

```ts
  options.testSequencer = getTestSequencer({
    rootDir: options.rootDir,
    testEnvironment: options. testSequencer || DEFAULT_CONFIG. testSequencer,
  });
```

Reply:
if so, also remove https://github.com/facebook/jest/blob/c85a5e9b1a8a82d837418d79bdb98493b4df460b/packages/jest-config/src/normalize.ts#L592-L602

Reply:
(might be a slippery slope, and should rather be part of a larger refactor of `normalize`)